src/templates/React InstantSearch Native/package.json

```diff
@@ -10,22 +10,23 @@
     "eject": "expo eject"
   },
   "dependencies": {
-    "algoliasearch": "3.32.1",
-    "expo": "32.0.6",
-    "prop-types": "15.7.2",
-    "react": "16.5.0",
-    "react-instantsearch-native": "{{libraryVersion}}",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz"
+    "algoliasearch": "4.11.0",
+    "expo": "44.0.3",
+    "prop-types": "15.8.0",
+    "react": "17.0.2",
+    "react-instantsearch-native": "6.18.0",
```

Comment:
This should remain "{{libraryVersion}}" so the user can pick their own.

Reply:
```suggestion
    "react-instantsearch-native": "{{libraryVersion}}",
```

Reply:
Thanks for noticing! Fixed in b76d49b.